### PR TITLE
Changed Mitre overview description 

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

- Mitre feature: changed its current description by `Explore security alerts mapped to adversary tactics and techniques for better threat understanding.`
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7027

### Evidence

image

### Test

- Check mitre overview description

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by guidomodarelli — COMMENTED** [review]
https://github.com/wazuh/wazuh-dashboard-plugins/blob/fc92c443b813e47bf09ef1b01cc77217023c5a06/plugins/main/common/wazuh-modules.ts#L112 

```diff
diff --git a/plugins/main/common/wazuh-modules.ts b/plugins/main/common/wazuh-modules.ts
index 341300f65..ec327ed57 100644
--- a/plugins/main/common/wazuh-modules.ts
+++ b/plugins/main/common/wazuh-modules.ts
@@ -109,7 +109,7 @@ export const WAZUH_MODULES = {
     title: 'MITRE ATT&CK',
     appId: 'mitre-attack',
     description:
-      'Security events from the knowledge base of adversary tactics and techniques based on real-world observations',
+      'Explore security alerts mapped to adversary tactics and techniques for better threat understanding',
   },
   syscollector: {
     title: 'Inventory data',
```

**Review by Machi3mfl — APPROVED** [review]
CR: ✅ 
Test: ✅ 

### Before

Screenshot 2024-09-30 at 2 20 00 PM


### After

Screenshot 2024-09-30 at 2 19 50 PM

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Update malware detection group values in data sources [#6963](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6963)
 - Changed the registration id of the Settings application for compatibility with OpenSearch Dashboard 2.16.0 [#6938](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6938)
 - Changed Malware detection dashboard visualizations [#6964](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6964)
+- Changed MITRE ATT%CK overview description [#7032] (https://github.com/wazuh/wazuh-dashboard-plugins/pull/7032)
```
Comment (asteriscos) [inline CHANGELOG.md:18]: Please change the `%` character to `&`

**Review by asteriscos — CHANGES_REQUESTED** [review]
CR: :red_circle:

**Review by asteriscos — APPROVED** [review]
LGTM :green_circle:

**Review by Machi3mfl — APPROVED** [review]

**Pull request merged**